project/plugins.sbt

```diff
@@ -1,29 +1,42 @@
+// This file contains a list of SBT plugins.
+
+// === resolvers
 resolvers += Classpaths.typesafeReleases
+resolvers += Classpaths.sbtPluginReleases
+resolvers += Resolver.sonatypeRepo("releases")
+resolvers += Resolver.sonatypeRepo("snapshots")
```

Comment:
It was added by me just for `addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.7.0-SNAPSHOT")`. But as we stated - this kind of plugin can be added to SBT's global plugins.